test/test_pipeline_baseline.py

```diff
@@ -184,13 +184,29 @@ def test_go_gapic_client_baseline(tmpdir):
                       tmpdir)
 
 
-def _test_java_baseline(task_name, test_name, tmpdir):
-    _test_baseline(task_name, test_name, 'java', str(tmpdir), {})
+def _test_java_baseline(task_name, test_name, tmpdir, extra_kwargs={}):
```

Comment:
[Don't use mutable defaults](http://docs.python-guide.org/en/latest/writing/gotchas/), which can lead to surprising behavior later. Instead do

```
def ...(..., extra_kwargs=None):
  extra_kwargs = extra_kwargs or {}
```